addon/components/denali-alert.js

```diff
@@ -28,12 +31,12 @@ export default class DenaliAlertComponent extends Component {
   }
 
   get iconClass() {
```

Comment:
Should we add an `if this.iconClass` to the template? Looked a little strange reading the test to see that the element still existed

Reply:
I originally had that, but it seems that denali relies on the element for spacing:

<img width="440" alt="Screen Shot 2021-02-12 at 10 43 50 AM" src="https://user-images.githubusercontent.com/2983409/107796279-47be4e00-6d1f-11eb-9c4c-b5be7344139d.png">